docs/diagrams/UiWorkflow.puml

```diff
@@ -0,0 +1,81 @@
+@startuml
+!include style.puml
+
+box Ui UI_COLOR_T1
+participant ":Browser" as Browser UI_COLOR
+participant AppSpot UI_COLOR
```

Comment:
This should be Google App Engine, if you're unable to display it as it's logo.

Reply:
Noted!